test/lit.cfg

```diff
@@ -2365,6 +2365,10 @@ if not hasattr(config, 'target_swift_reflection_test'):
 config.substitutions.append(('%target-swift-reflection-test', config.target_swift_reflection_test))
 
 config.substitutions.append(('%target-swift-reflection-dump', '{} {} {}'.format(config.swift_reflection_dump, '-arch', run_cpu)))
+
+config.substitutions.append(('%target-swiftc_driver-emit-module-interface\(([^)]+)\)',
+                             SubstituteCaptures(r'%s -enable-library-evolution -verify-emitted-module-interface -emit-module-interface-path \1' % (
+                                 escape_for_substitute_captures(config.target_swiftc_driver)))))
```

Comment:
Should we add `-swift-version 5` too? It's required with `-enable-library-evolution`.

Reply:
Yeah, good catch. That seems like the default we want.

Reply:
Updated